src/airflow/airqo_etl_utils/schema/mobile_measurements.json

```diff
@@ -52,12 +57,22 @@
     "mode": "NULLABLE",
     "description": "μg/m3."
   },
+  {
+    "name": "pm10_raw_value_aqi",
+    "type": "STRING",
+    "mode": "NULLABLE"
+  },
   {
     "name": "no2_raw_value",
     "type": "FLOAT",
     "mode": "NULLABLE",
     "description": "ppb."
   },
+  {
+    "name": "no2_raw_value_aqi",
+    "type": "STRING",
+    "mode": "NULLABLE"
+  },
```

Comment:
@sserurich field for no2 AQI

Reply:
Brilliant @NoahNsimbe ...thanks for this. I am wondering if this aligns with our direction of assigning AQI values only to averaged measurements. Are we shifting goal posts just because of one Client? cc: @sserurich